### ISSUE TITLE
Let users know which fields are mandatory in the feedback form

### ISSUE THREAD

**Issue by willdurand** [mozilla/addons-frontend#12847]
This is a follow-up to https://github.com/mozilla/addons-frontend/issues/12841

### Describe the problem and steps to reproduce it:

When submitting a feedback form (for add-ons, collections, users or ratings), the "submit" button is disabled until all required fields are set but this is bad UX since it's hard to grasp that the fields not explicitly marked as "optional" are "required".

### What did you expect to happen?

We should have feedback on mandatory fields that are not set.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDFRNT-330)

**Comment by alexandruschek** [comment]
Hello, 

I tested this on -STAGE and I noticed that the user is not able to submit the form unless they completed every necessary step, and feedback appears for every part of the form that is not completed(and is mandatory), so it looks good.
I noticed only this thing, the "little grey box" does not stick to the field that it refers to if I scroll up or down(this applies for every field):
<img width="1728" alt="Screenshot 2024-02-21 at 10 44 04 AM" src="https://github.com/mozilla/addons-frontend/assets/139126091/77c9f590-e879-4d64-8568-c4c07a2949fa">
The report continues to get sent as expected and gets into Cinder.

Thank you!
